doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+================================================
+Tutorial: Linear algebra on n-dimensional arrays
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+Prerequisites
+=============
```

Comment:
minor: these headings will end up in ToC's and generate links. should it be changed to just bold font to avoid that?